src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+@Entity
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
     private Country country;
 
     public Actor() {
```

Comment:
Don't create a default constructor when it is not needed. Since you have a constructor with parameters, the default constructor is not required unless explicitly used by Hibernate or other frameworks.